src/switchroot/ostree-remount.c

```diff
@@ -167,15 +167,6 @@ main (int argc, char *argv[])
   }
   g_autoptr (GVariantDict) ostree_run_metadata = g_variant_dict_new (ostree_run_metadata_v);
 
-  /* The /sysroot mount needs to be private to avoid having a mount for e.g. /var/cache
```

Comment:
It is very compelling to be able to delete this code...longer term what I think we want is for `ostree-remount.c` to basically go away.